src/app/components/navbar/navbar.controller.js

```diff
@@ -25,4 +38,14 @@ angular.module('transmartBaseUi')
                 }
             });
 
-        }]);
+        }])
+    .config(function (IdleProvider, KeepaliveProvider) {
+        IdleProvider.idle(20*60); // 20min in seconds
```

Comment:
You know that hardcoding is bad practice. We've seen to much in this project. 20 * 60. what is that? and then followed by a narration to explain it is not pretty. on top of that, Controller ideally should not be the place to configure something.

Better approach:

- Config file
- Constants 

`
idleProvider.idle(IDLE_IN_MINUTES*60);
`